DashKitUI/Views/PodSetupView.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  PodSetupView.swift
+//  DashKitUI
+//
+//  Created by Pete Schwamb on 5/17/21.
+//  Copyright © 2021 Tidepool. All rights reserved.
+//
+
+import SwiftUI
+import LoopKitUI
+
+struct PodSetupView: View {
+    @Environment(\.dismiss) private var dismiss
+    
+    private struct AlertIdentifier: Identifiable {
+        enum Choice {
+            case skipOnboarding
+        }
+        var id: Choice
+    }
+    @State private var alertIdentifier: AlertIdentifier?
+
+    let nextAction: () -> Void
+    let allowDebugFeatures: Bool
+    let skipOnboarding: () -> Void
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            close
+            ScrollView {
+                content
+            }
+            Spacer()
+            continueButton
+                .padding(.bottom)
+        }
+        .padding(.horizontal)
+        .navigationBarTitle("")
+        .navigationBarHidden(true)
+        .alert(item: $alertIdentifier) { alert in
+            switch alert.id {
+            case .skipOnboarding:
+                return skipOnboardingAlert
+            }
+        }
+    }
+    
+    @ViewBuilder
+    private var close: some View {
+        HStack {
+            Spacer()
+            closeButton
+        }
+        .padding(.top)
+    }
+        
+    @ViewBuilder
+    private var content: some View {
+        VStack(alignment: .leading, spacing: 2) {
+            title
+                .padding(.top, 5)
+                .onLongPressGesture(minimumDuration: 2) {
+                    didLongPressOnTitle()
+                }
+            Divider()
+            bodyText
+                .foregroundColor(.secondary)
+                .padding(.top)
+        }
+    }
+
+    @ViewBuilder
+    private var title: some View {
+        Text("Pod Setup")
+            .font(.largeTitle)
+            .bold()
+            .padding(.vertical)
+    }
+    
+    @ViewBuilder
+    private var bodyText: some View {
+        Text("You will now begin the process of configuring your reminders, filling your Pod with insulin, pairing to to your device and placing it on your body.")
+    }
+    
+    private var closeButton: some View {
+        Button(LocalizedString("Close", comment: "Close button title"), action: {
+            self.dismiss()
+        })
+    }
+
+    private var continueButton: some View {
+        Button("Continue", action: nextAction)
+            .buttonStyle(ActionButtonStyle())
+    }
+    
+    private var skipOnboardingAlert: Alert {
+        Alert(title: Text("Skip Omnipod Onboarding?"),
+              message: Text("Are you sure you want to skip Omnipod Onboarding?"),
+              primaryButton: .cancel(),
+              secondaryButton: .destructive(Text("Yes"), action: skipOnboarding))
+    }
```

Comment:
Localize?

Reply:
Probably shouldn't put debug UI into localization.

Reply:
Oh, I see, the "Continue" button. Thanks.